src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -453,4 +460,60 @@ default String speedrunPBMessage() {
     default String speedrunMessage() {
         return "%USERNAME% has just finished a speedrun of %QUEST% with a time of %TIME% (their PB is %BEST%)";
     }
+
+    @ConfigItem(
+        keyName = "killCountEnabled",
+        name = "Enable Kill Count",
+        description = "Enable notifications for boss kill count milestones",
+        position = 37,
+        section = killCountSection
+    )
+    default boolean notifyKillCount() {
+        return false;
+    }
+
+    @ConfigItem(
+        keyName = "killCountSendImage",
+        name = "Send Image",
+        description = "Send image with the notification",
+        position = 38,
+        section = killCountSection
+    )
+    default boolean killCountSendImage() {
+        return true;
```

Comment:
```suggestion
        return false;
```
Should be disabled by default to be inline with the other notifs, also an issue with our speedRun notification but that's another PR.